public/index.html

```diff
@@ -1,16 +1,33 @@
 <!DOCTYPE html>
-<html lang="">
+<html lang="en">
   <head>
     <meta charset="utf-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="theme-color" content="#252529">
     <meta name="viewport" content="width=device-width,initial-scale=1.0" />
+    <meta
+      name="description"
+      content="Torus Wallet is the easiest solana digital wallet available on the web.
```

Comment:
Maybe we can use `Solana Wallet`? The naming in the docs are
- [Solana Wallet](https://docs.tor.us/solana-wallet/get-started)
- [Torus Solana Wallet](https://docs.tor.us/guides/torus-solana-wallet-intergration) (old)

Reply:
how about solana wallet by torus sounds ?
we need to include torus right some where...